clowdapp.yaml

```diff
@@ -42,10 +34,42 @@ objects:
       enabled: true
       appName: "compliance"
       insightsOnly: true
-    jobs:
+    deployments:
+    - name: db-migrations
+      podSpec:
+        image: ${IMAGE}:${IMAGE_TAG}
+        initContainers:
+        - command:
+            - bash
+            - -c
+            - |
+              bundle exec rake --trace db:debug db:migrate
+          inheritEnv: true
+        resources:
+          limits:
+            cpu: ${CPU_LIMIT_SIDE}
+            memory: ${MEMORY_LIMIT_SIDE}
+          requests:
+            cpu: ${CPU_REQUEST_SIDE}
+            memory: ${MEMORY_REQUEST_SIDE}
+        env:
+        - name: APPLICATION_TYPE
+          value: sleep
```

Comment:
This used to run as part of the prometheus_exporter, so part of this PR is to rearchitect the migrations a bit. The clowder team is currently investigating how to run migrations without having $REPLICAS processes running at the same time. For now, I've updated the entrypoint.sh to include a sleep APPLICATION_TYPE that does not ever terminate the pod.

We'll be replacing the import-ssg logic with a CronJob like we discussed - that change is not part of this PR!